docs/install/rancher.md

```diff
@@ -1,248 +1,159 @@
-For production installations, follow our [production](/docs/prerequisites/etcd#production) recipe.
-Make a note of the etcd endpoint URL in either case.
+* [kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl)
 
-1. Select the `System` project of your cluster
+> ⚠️ Make sure to add an [Ondat licence](/docs/operations/licensing/) after installing.
 
-    ![install-1](/images/docs/rancher-ui-v2/rancher-1.png)
+> ⚠️ Make sure you have a running RKE cluster with a minimum of 3 worker nodes and the sufficient [Role-Based Access Control (RBAC)](https://kubernetes.io/docs/reference/access-authn-authz/rbac/) permissions to deploy and manage applications in the cluster.
 
-2. Select the `Apps` tab and click `Launch`
+> ⚠️ Make sure your RKE cluster uses a Linux distribution that is officially supported by Rancher as your node operating system and has the required LinuxIO related kernel modules are available for Ondat to run successfully. A strong recommendation would be to review [SUSE Rancher Support Matrix](https://www.suse.com/suse-rancher/support-matrix/all-supported-versions/) documentation to ensure that you are using a supported Linux distribution.
 
-    ![install-2](/images/docs/rancher-ui-v2/rancher-2.png)
+## Procedure
 
-3. Search for Ondat and click on the App
+### Option A - Using Ondat Kubectl Plugin
 
-    ![install-3](/images/docs/rancher-ui-v2/rancher-3.png)
+#### Step 1 - Install Ondat Kubectl Plugin
 
-    This will install the Ondat operator, which manages the Ondat
-    DaemonSet.
+* Ensure that the Ondat kubectl plugin is installed on your local machine and is available in your `$PATH`:
+  * [kubectl-storageos](/docs/reference/kubectl-plugin/)
 
-4. Check and ammend installation options
+#### Step 2 - Install Local Path Provisioner
 
-    A generic configuration for Ondat is preset using the default values in
-    the form. Be sure to check the etcd address and ensure it matches the value
-    you noted at the beginning of this guide.
+1. By default, a newly provisioned RKE cluster does not have any CSI driver deployed. Run the following commands against the cluster to deploy a [Local Path Provisioner](https://github.com/rancher/local-path-provisioner) to provide local storage for Ondat's embedded `etcd` cluster operator deployment.
 
-    The catalog form exposes several useful parameters - documented
-    [below](/docs/install/rancher#simplecustomization}).
+```bash
+kubectl apply --filename="https://raw.githubusercontent.com/rancher/local-path-provisioner/v0.0.21/deploy/local-path-storage.yaml"
+```
 
-    For further customization, you can opt to set the option to 'Install
-    Ondat Cluster' to false and install a custom CR. See [below](/docs/install/rancher#advancedcustomization) for this.
+2. Define and export the `ETCD_STORAGECLASS` environment variable so that value is `local-path`, which is the default StorageClass name for the Local Path Provisioner.
```

Comment:
I'd personally use the list format of  1. 1. 1. 1. 
```
1. x
1. y
1. z
```

So in case of alterations to the steps in time, we don't have to change all the subsequent numbers by hand. That requires that the markdown inside the list is indented to keep the markdown engine to render the numbers correctly.

Reply:
I have made the changes suggested.